gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Mon Jul 22 17:43:50 PDT 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+networkTimeout=10000
```

Comment:
What is this for? This timeout value seems too small?

Reply:
This was added automatically by Gradle when I ran the wrapper task. I'll remove it